packages/quantic/cypress/integration/tab/tab.cypress.ts

```diff
@@ -6,6 +6,7 @@ import {
 } from '../../page-objects/search';
 import {TabExpectations as Expect} from './tab-expectations';
 import {TabActions as Actions} from './tab-actions';
+import {TabSelector, TabSelectors} from './tab-selectors';
```

Comment:
See my comment on tab-selectors.ts don't create a new selector for the button, just modify the tab selector to select the tab after your changes.

Reply:
selectors modified